kittengroomer/helpers.py

```diff
@@ -43,82 +30,44 @@ def __init__(self, src_path, dst_path):
         Create various properties and determine the file's mimetype.
         """
         self.src_path = src_path
-        self.dst_path = dst_path
-        self.filename = os.path.basename(self.src_path)
-        self._file_props = {
-            'filepath': self.src_path,
-            'filename': self.filename,
-            'file_size': self.size,
-            'maintype': None,
-            'subtype': None,
-            'extension': None,
-            'safety_category': None,
-            'symlink': False,
-            'copied': False,
-            'description_string': [],  # array of descriptions to be joined
-            'errors': {},
-            'user_defined': {}
-        }
-        self.extension = self._determine_extension()
-        self.set_property('extension', self.extension)
-        self.mimetype = self._determine_mimetype()
+        self.dst_dir = os.path.dirname(dst_path)
+        self.filename = os.path.basename(src_path)
+        self.size = self._get_size(src_path)
+        self.is_dangerous = False  # Should this be some other value initially?
```

Comment:
For binary values, I'd keep it to True or False. Would it make sense to put it to True by default?